test/rekt/features/channel/features.go

```diff
@@ -191,12 +193,28 @@ func AsyncHandlerUpdate(createSubscriberFn func(ref *duckv1.KReference, uri stri
 	f.Setup("channel is ready", channel_impl.IsReady(name))
 	f.Setup("subscription is ready", subscription.IsReady(sub))
 
-	f.Requirement("update channel async handler", channel_impl.Install(name, channel_impl.WithAnnotations(map[string]interface{}{
```

Comment:
looking at the error again:

```
    channel_impl.go:92: failed to update resource admission webhook "validation.inmemorychannel.eventing.knative.dev" denied the request: validation failed: Channel.Spec.Subscribers changed by user prow-job@knative-tests.iam.gserviceaccount.com which was not the system:serviceaccount:knative-eventing:eventing-controller service account: spec.subscribers
        {[]v1.SubscriberSpec}:
        	-: "[{Name:0xc0009a5ea0 UID:828704ea-0f50-48ab-bcd7-9b9cacc19796 Generation:1 SubscriberURI:http://sink-dwarkbpn.test-wcdvjxod.svc.cluster.local/ SubscriberCACerts: SubscriberAudience: ReplyURI: ReplyCACerts: ReplyAudience: Delivery: Auth:}]"
        	+: "[]"
```

I saw that the subscribers are "replaced" w/ nothing (`+:"[]"`), which lead me to the fact that we here just re-install the channel_impl, based from its manifest `yaml` file, which lacks the actual `subscribers`